### PR TITLE
Allow decryption of empty values

### LINKED ISSUE #138: go-sops fails to decrypt JSON file with empty value
`example.json` contains an empty value that go-sops fails to decrypt with the error:
```
Could not load file: Error decrypting tree: Error walking tree: Could not decrypt value: Input string  does not match sops' data format
```

### PR DESCRIPTION
**Pull request by autrilla**
Fixes #138

### CODE REVIEW

File: aes/decryptor.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type stashData struct {
 // Cipher encrypts and decrypts data keys with AES GCM 256
 type Cipher struct{}
 
-var encre = regexp.MustCompile(`^ENC\[AES256_GCM,data:(.+),iv:(.+),tag:(.+),type:(.+)\]`)
+var encre = regexp.MustCompile(`^ENC\[AES256_GCM,data:(.*),iv:(.+),tag:(.+),type:(.+)\]`)
```
Comment (autrilla) [inline aes/decryptor.go]: sops-python stores an empty value, so that's the route we have to go.

Comment (jvehent) [inline aes/decryptor.go]: Fair. I'm ok to merge this.

**Comment by jvehent** [comment]
Looks good. Might I ask for a test? :)

**Comment by autrilla** [comment]
@jvehent there's already one. Look at the change in aes/decryptor_test.go

**Pull request merged**